performance-tests/bench_2/builder/ListenerFactory.cpp

```diff
@@ -0,0 +1,73 @@
+#include "ListenerFactory.h"
+
+namespace Builder {
+
+Bench_Builder_Export
+void RegisterDomainParticipantListener(const std::string& name, const std::function<DDS::DomainParticipantListener::_var_type(const Builder::PropertySeq&)>& factory)
```

Comment:
Why are these function names capitalized?

Reply:
Hmm, they started life as non-templated classes that inherited from the appropriate template specialization, but then I changed my mind and just made them functions. I'll rename them to use snake case.